zigpy/zcl/clusters/general.py

```diff
@@ -942,6 +942,27 @@ class ServerCommandDefs(BaseCommandDefs):
             direction=Direction.Client_to_Server,
         )
 
+    class ClientCommandDefs(BaseCommandDefs):
+        off: Final = ZCLCommandDef(
+            id=0x00, schema={}, direction=Direction.Server_to_Client
+        )
+        on: Final = ZCLCommandDef(
+            id=0x01, schema={}, direction=Direction.Server_to_Client
+        )
+        toggle: Final = ZCLCommandDef(
+            id=0x02, schema={}, direction=Direction.Server_to_Client
+        )
```

Comment:
From what I understand, these commands can be sent in both directions, so I am adding them here. They aren't really required for this PR to function, any guidance is appreciated

Reply:
These commands really can only go in one direction: a lightbulb, for example, is a server and a remote is a client. The remote sends an `OnOff:toggle()` command to the lightbulb and the lightbulb turns on.